<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CereStimDBS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>721</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0,0,0">
     <item row="4" column="1">
      <widget class="QLabel" name="amp_label">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Amplitude (uA): </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="status_label">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Status </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="ramp_label">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ramp Dur. (s): </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QSpinBox" name="elec_spinBox">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>96</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="dur_label">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Final Dur. (s): </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="connect_pushButton">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="dur_doubleSpinBox">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>30.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="ramp_doubleSpinBox">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="maximum">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="start_pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="width_label">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Width (uS): </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QComboBox" name="interphase_comboBox">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Min Sep</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Max Sep</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="elec_label">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Electrode: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="generate_pushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="amp_spinBox">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="device_comboBox">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="interphase_label">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Interphase: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <layout class="QVBoxLayout" name="status_verticalLayout"/>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="polarity_comboBox">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Cathodic First</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Anodic First</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cathodic-Mono</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Anodic-Mono</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QSpinBox" name="width_spinBox">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <number>44</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="polarity_label">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Polarity: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="freq_label">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Frequency (Hz): </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="refresh_pushButton">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QWidget" name="plot_widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QSpinBox" name="freq_spinBox">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>refresh_pushButton</tabstop>
  <tabstop>device_comboBox</tabstop>
  <tabstop>connect_pushButton</tabstop>
  <tabstop>polarity_comboBox</tabstop>
  <tabstop>dur_doubleSpinBox</tabstop>
  <tabstop>ramp_doubleSpinBox</tabstop>
  <tabstop>amp_spinBox</tabstop>
  <tabstop>interphase_comboBox</tabstop>
  <tabstop>width_spinBox</tabstop>
  <tabstop>freq_spinBox</tabstop>
  <tabstop>elec_spinBox</tabstop>
  <tabstop>generate_pushButton</tabstop>
  <tabstop>start_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
